ppa/archive/models.py

```diff
@@ -442,6 +442,14 @@ def __str__(self):
             return "%s (%s)" % (self.source_id, self.pages_digital)
         return self.source_id
 
+    @property
+    def cluster_id_s(self):
```

Comment:
I would just name the property `cluster_id` — the `_s` suffix is used for solr dynamic fields, not meaningful here

Reply:
Is `.cluster_id` reserved for the numerical django ID for the cluster object, though? That's what seems to be set there on digiworks now.

Reply:
is it? I would expect that to be at `.cluster.pk` on the model  — but you've been looking at the code more recently than me!  But maybe a different name would be better — include index somehow to differentiate? `index_cluster_id`

the other `.cluster_id` that you had the commented question on is the object returned by Solr

Reply:
`index_cluster_id` it is! thanks!